src/atama/receiver/node/receiver_node.cpp

```diff
@@ -39,16 +39,17 @@ ReceiverNode::ReceiverNode(rclcpp::Node::SharedPtr node, std::shared_ptr<atama::
 {
   get_joints_client = node->create_client<tachimawari_interfaces::srv::GetJoints>(
     "/joint/get_joints");
-  
+
   get_orientation_subsciber = node->create_subscription<kansei_interfaces::msg::Orientation>(
     "measurement/orientation", 10,
     [this](const kansei_interfaces::msg::Orientation::SharedPtr message) {
       // this->head->set_yaw(message->orientation[2]);
     }
   );
 
-  get_detection_result_subsciber = node->create_subscription<ninshiki_interfaces::msg::DetectedObjects>(
-    "ninshiki_py/detection", 10, 
+  get_detection_result_subsciber =
+    node->create_subscription<ninshiki_interfaces::msg::DetectedObjects>(
```

Comment:
that's why i suggest to alias some message, just to prevent writing long declaration for the message struct (like `ninshiki_interfaces::msg::DetectedObjects` above).

Reply:
done